microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -134,37 +152,45 @@ You are paused on `A`.  After you click **Step into**, DevTools runs this line o
 
```

Comment:
151 change heading from:
Step out of line of code
to:
Step out of a line of code
or:
Step out from a line of code